Moovies/Moovies/MainModule/ViewModel/MainViewModel.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  MainViewModel.swift
+//  Moovies
+//
+//  Created by Vera Zaitseva on 11.10.2021.
```

Comment:
```suggestion
//  Created by Vera Zaitseva on 11.10.2021.
```
ставь линтер и форматер